pkgs/misc/drivers/moltengamepad/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, fetchFromGitHub, libudev }:
+
+stdenv.mkDerivation rec {
+  name = "moltengamepad-git-${version}";
+  version = "2016-05-04";
+
+  src = fetchFromGitHub {
+    owner = "jgeumlek";
+    repo = "MoltenGamepad";
+    rev = "6656357964c22be97227fc5353b53c6ab1e69929";
+    sha256 = "05cpxfzxgm86kxx0a9f76bshjwpz9w1g8bn30ib1i5a3fv7bmirl";
+  };
+
+  buildInputs = [ libudev ];
+
+  buildPhase = ''
+    mkdir -p $out/bin
```

Comment:
Is this directory need for `make`? If not, I think this command would make more sense in the `installPhase`.